src/module/address/domain/address.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { Entity, Column, OneToOne } from 'typeorm';
+import { Base } from 'src/common/domain/base.entity';
+
+@Entity({ name: 'address'})
+export class Address extends Base {
+
+  @Column({ nullable: true })
+  country: string;
+
+  @Column({ nullable: true })
+  state: string;
+
+  @Column({ nullable: true })
+  city: string;
+  
+  @Column({ type: 'json', nullable: true, default: '[]'})
```

Comment:
type json is not commonly used, is unsafe. if you want to follow this design you should be creating another entity and making the relation.

Reply:
done